### PR TITLE
[vis] render line breaks in TableViz

### LINKED ISSUE #1785: [vis] render line breaks in table vis
- render `\n` as line breaks in table vis by default

### PR DESCRIPTION
**Pull request by ascott**
wrap values in a class that uses `white-space: pre`, to honor line breaks, tabs, spaces.
http://stackoverflow.com/questions/219219/how-to-change-a-span-to-look-like-a-pre-with-css

fixes: https://github.com/airbnb/superset/issues/1785

before:
screenshot 2017-02-10 09 34 21

after:
screenshot 2017-02-10 09 30 25

plz review @airbnb/superset-reviewers

### CODE REVIEW

File: superset/viz.py [hunk 1]
```diff
@@ -597,7 +601,13 @@ def get_data(self):
             aggfunc=self.form_data.get('pandas_aggfunc'),
             margins=True,
         )
+
+        # replace line breaks(\n) with <br>
+        for col in df.columns:
+            df[col] = df[col].astype(np.str).str.replace('\n', '<br>')
+
         return df.to_html(
+            escape=False,
```
Comment (mistercrunch) [inline superset/viz.py]: wouldn't that allow for `<script/>`? , that seems like a security concern

Comment (ascott) [inline superset/viz.py]: good point, i will check on that.

Comment (ascott) [inline superset/viz.py]: Tested this and checked the docs. Looks like the DataTables javascript renderer disallows xss attacks via this vector. 
https://datatables.net/manual/security#Cross-Site-Scripting

screenshot 2017-02-06 10 58 33

Comment (ascott) [inline superset/viz.py]: ah, nevermind, missed something here. `escape=False` does allow `<script></script>` tags. will fix.

**Comment by ascott** [comment]
@mistercrunch thanks for the review. decided to only replace line breaks for table viz for now since that is the main use case policy was needing. was going to port to pivot table too but going to leave that out for now since we don't have a use case and it's slightly more involved for pivot tables.

**Comment by ascott** [comment]
@mistercrunch i addressed your comments, can you PTAL? 🎉

**Comment by mistercrunch** [comment]
that could be quite a bit of data going through a regex (might not matter but may be missing tabs and multiple spaces and such), what about wrapping in <pre/>?

**Comment by mistercrunch** [comment]
Looks like there's a way to do it in pure CSS too without adding a tag:
http://stackoverflow.com/questions/219219/how-to-change-a-span-to-look-like-a-pre-with-css

**Comment by ascott** [comment]
so simple! that works great, thanks @mistercrunch!

**Comment by mistercrunch** [comment]
Sweet. LGTM!

**Comment by ascott** [comment]
just tested this on staging and noticed that with the <pre> formatting really long lines make the table cell really wide and then you have to scroll the table horiztontally.

table-pre

will submit a new pr to fix this.

**Comment by lilila** [comment]
I think this modification created a new problem: Whenever I want to filter by clicking on one table cell I get the following filter query :  AND genre IN ('<span class="like-pre">Drame</span>')

**Comment by mistercrunch** [comment]
@lilila looks like we overlooked that feature, I'll push a fix

**Pull request merged**